completions/bash/multipass

```diff
@@ -388,7 +388,12 @@ _multipass_complete()
     fi
 
     if [[ -n "${opts}" ]]; then
-        COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+        for word in "${COMP_WORDS[@]}"; do
+            if [[ "${COMP_LINE}" == *"${word} "* ]]; then
+                opts=("${opts[@]/$word}")
```

Comment:
Hey @sharder996, sorry, I am afraid this won't do because it can remove strings where it shouldn't. For instance, I have instances called `a` and `b`. Here's what completion does:

```
$ multipass info [TAB][TAB]
a          --all      b          --format   --help     --verbose
$ multipass info a [TAB][TAB]
b          --format   --help     --ll       --verbose
```

It removed all "a"s from `opts`, so `--all` became `--ll`...

Reply:
I think this would work:

```suggestion
                opts=("${opts[@]/\b$word\b}")
```